srcpkgs/swirc/template

```diff
@@ -1,23 +1,23 @@
 # Template file for 'swirc'
 pkgname=swirc
-version=3.3.9
+version=3.4.0
 revision=1
 build_style=configure
 configure_args="$(vopt_with notify libnotify)"
 make_build_args="PREFIX=/usr"
 make_check_args="PREFIX=/usr"
 make_install_args="PREFIX=/usr"
 hostmakedepends="gettext-devel-tools pkg-config which"
-makedepends="gettext-devel libcurl-devel libidn-devel ncurses-devel openssl-devel
- $(vopt_if notify libnotify-devel)"
+makedepends="gettext-devel hunspell-devel libcurl-devel libidn-devel
+ ncurses-devel openssl-devel $(vopt_if notify libnotify-devel)"
 checkdepends="cmocka-devel"
 short_desc="Curses ICB and IRC client"
 maintainer="Markus Uhlin <markus.uhlin@bredband.net>"
 license="BSD-3-Clause, ISC, MIT"
 homepage="https://www.nifty-networks.net/swirc"
 changelog="https://raw.githubusercontent.com/uhlin/swirc/master/CHANGELOG.md"
 distfiles="https://www.nifty-networks.net/swirc/releases/${pkgname}-${version}.tgz"
```

Comment:
```suggestion
distfiles="https://www.nifty-networks.net/swirc/releases/swirc-${version}.tgz"
```

Reply:
Can I commit the change here at GitHub or shall I do a force push? (There's a button "commit suggestion".)

Reply:
force push is probably less steps

Reply:
Ok, done